hack/jenkins/jobs/bootkube_conformance_cncf.groovy

```diff
@@ -0,0 +1,34 @@
+// META
+repo = "kubernetes-incubator/bootkube"
+
+// CONFIG
+org_whitelist = ['coreos', 'coreos-inc']
+job_admins = ['ericchiang', 'rithujohn191', 'rphillips']
+user_whitelist = job_admins
+
+// JOBS
+job_name = "tku-bootkube-conformance-cncf"
+
+pipelineJob(job_name) {
+  parameters {
+    stringParam('sha1', 'origin/master', 'git reference to build')
+  }
+  definition {
+    cpsScm {
+      scm {
+        git {
+          remote {
+            github("${repo}")
+            refspec('+refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*')
+            credentials('github_userpass')
+          }
+          branch('${sha1}')
+        }
+      }
+      scriptPath('hack/jenkins/pipelines/bootkube-conformance/Jenkinsfile')
+    }
+    triggers {
+        cron('H 11 * * *')
```

Comment:
nit: 2 spaces everywhere else in the file *shrug*

Reply:
done.